packages/ui/src/ui/components/Dialog/controls/PoolSuggestControl/PoolSuggestControl.tsx

```diff
@@ -20,6 +20,7 @@ type Props = DialogControlProps<string> & {
     cluster?: string;
     // If not defined then default pool tree should be used
     poolTree?: string;
+    setOnLoadPoolsValue?: (poolNames: string[]) => string;
```

Comment:
I am not sure, but may be it is possible to make up a better name:
`calculateValueFromLoaded`/`calculateValueFromLoadedItems`/`recalculateValueOnLoad`/...
```ts
calculateValueFromLoaded({loadedPoolNames}: {loadedPoolNames: Array<string>})
```
What do you think?
And may be it is better to pass `cluster` as a parameter to this callback instead of using closures.

Reply:
> I am not sure, but may be it is possible to make up a better name: `calculateValueFromLoaded`/`calculateValueFromLoadedItems`/`recalculateValueOnLoad`/...
> 
> ```ts
> calculateValueFromLoaded({loadedPoolNames}: {loadedPoolNames: Array<string>})
> ```
> 
> What do you think? And may be it is better to pass `cluster` as a parameter to this callback instead of using closures.

I'll try to find better name 👍🏻

We pass _destination_ cluster name to `PoolSuggestControl` prop, and if we want to return _current_ cluster name as callback parameter we could:
 - connect `PoolSuggestControl` to the store, to get current cluster name inside control component
 - or add one more prop `currentCluster` to return it back with callback
I think these two options are more complicated and hide part of the logic inside control 🤔